src/main/java/net/coderbot/iris/uniforms/CameraUniforms.java

```diff
@@ -27,7 +27,7 @@ public static void addCameraUniforms(UniformHolder uniforms, FrameUpdateNotifier
 			.uniform3d(PER_FRAME, "previousCameraPosition", new PreviousCameraPosition(notifier));
 	}
 
-	private static int getRenderDistanceInBlocks() {
+	public static int getRenderDistanceInBlocks() {
 		return client.options.viewDistance * 16;
```

Comment:
```diff
- return client.options.viewDistance * 16;
+ return client.gameRenderer.getViewDistance();
```
Use `GameRenderer#getViewDistance`

Reply:
this should be something for another pr, as this is the current behavior of CameraUniforms.